packages/contentful/src/gp2/queries/event.queries.ts

```diff
@@ -2,6 +2,34 @@
 
 import { gql } from 'graphql-tag';
 
+export const eventRelatedOutputQueryFragment = gql`
+  fragment EventRelatedOutputData on Outputs {
+    sys {
+      id
+    }
+    title
+    documentType
+    type
+    relatedEntitiesCollection(limit: 1) {
+      items {
+        __typename
+        ... on WorkingGroups {
+          sys {
+            id
+          }
+          title
+        }
+        ... on Projects {
+          sys {
+            id
+          }
+          title
+        }
+      }
+    }
+  }
+`;
```

Comment:
I think we can import `relatedOutputQueryFragment` to this file instead of creating the same fragment 😉